packages/core/docs/shopify/api-client-reference.md

```diff
@@ -0,0 +1,13542 @@
+
```

Comment:
Seems like link to this file is not added to the sidebar in `packages/core/docs/.vuepress/config.js`

Reply:
Links Updated. Thanks.